pkgs/development/python-modules/confuse/default.nix

```
@@ -0,0 +1,25 @@
+{ buildPythonPackage
+, enum34
+, fetchPypi
+, lib
+, pythonOlder
+, pythonPackages
+}:
+buildPythonPackage rec {
+  pname = "confuse";
+  version = "1.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "07rbsd9l5widsirbn6rcfadh7gb2bdy2frhlga7bhxdizmhir2pn";
+  };
+
+  propagatedBuildInputs = [ pythonPackages.pyyaml ] ++ lib.optional (pythonOlder "3.4") enum34;
```

there will never be a <3.4 interpreter other than 2.7, and python2.7 is disabled anyway
```suggestion
  propagatedBuildInputs = [ pyyaml ];
```